recipes/libtool/all/conanfile.py

```diff
@@ -47,21 +61,26 @@ def _build_context(self):
             else:
                 yield
 
+    @property
+    def _datarootdir(self):
+        return os.path.join(self.package_folder, "bin", "share")
+
+    def _my_unix_path(self, path):
+        if tools.os_info.is_windows:
+            return tools.unix_path(path)
+        else:
+            return path
```

Comment:
This will be fixed in next Conan 1.25 release. unix_path() will have no effect on non-windows OSs

Reply:
What is the ETA of 1.25?

Reply:
this week

Reply:
Removed this method.